example/Makefile

```diff
@@ -24,8 +24,7 @@ down: .terraform env.auto.tfvars
 	tofu destroy -auto-approve
 
 port-forward:
-	source files/env.sh
-	bash files/registry-port-forward.sh
+	source files/env.sh && bash files/registry-port-forward.sh
```

Comment:
Is `.ONESHELL` broken on macos?

Reply:
.ONESHELL is not POSIX compliant, so it might be the reason why the implementation on macOS ignores it.

Could you please remove the .ONESHELL directive as well ? We should rather use a single technique to keep the env var around.

Reply:
Removed `.ONESHELL` in [66cc606](https://github.com/hetznercloud/kubernetes-dev-env/pull/19/commits/66cc606ec598db05716e1cffb9fccfad93f17711)